libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationConstants.java

```diff
@@ -123,6 +123,12 @@ private NavigationConstants() {
    */
   static final int NAVIGATION_LOCATION_ENGINE_INTERVAL_LAG = 1500;
 
+  static final float NAVIGATION_OFF_ROUTE_THRESHOLD = 50.0f;
```

Comment:
shouldn't it has suffix like `_METERS`? I mean does we have meters only or we use some others units(like cm or feet). Is it obvious that `50.0` is in meters?

Reply:
We're not adding it for the rest of constants, as they're internal we know that in this case is in meters.